cfn-resources/org-invitation/docs/README.md

```diff
@@ -51,7 +51,7 @@ Properties:
 
 #### Profile
 
-The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).
+Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used
```

Comment:
I think we need to have link to example secret profile setup

Reply:
Agree. We have a mix of both descriptions in different resources, but given that this description is used for the generated markdown docs linking to secret profile setup seems more helpful.